playbook-website/app/javascript/site_styles/_visual_guidelines.scss

```diff
@@ -31,14 +36,96 @@
       }
     }
 
-    .two { margin-left: 50px; }
-    .three { margin-left: 100px; }
-    .four { margin-left: 150px; }
-    .five { margin-left: 200px; }
-    .six { margin-left: 250px; }
-    .seven { margin-left: 300px; }
-    .eight { margin-left: 350px; }
-    .nine { margin-left: 400px; }
-    .ten { margin-left: 450px; }
+    .two {
+      margin-left: 50px;
+    }
+
+    .three {
+      margin-left: 100px;
+    }
+
+    .four {
+      margin-left: 150px;
+    }
+
+    .five {
+      margin-left: 200px;
+    }
+
+    .six {
+      margin-left: 250px;
+    }
+
+    .seven {
+      margin-left: 300px;
+    }
+
+    .eight {
+      margin-left: 350px;
+    }
+
+    .nine {
+      margin-left: 400px;
+    }
+
+    .ten {
+      margin-left: 450px;
+    }
+  }
+
+  ul.opacity-removing-utilities {
+
+    &.opacity-removing-utilities {
+      list-style: none;
+      display: flex;
```

Comment:
Doesn't this exist in the other file above?

Reply:
Yes, as I said in my notes: I made an opacity_guidelines.scss but it would not connect to the opacity.html.erb and only worked when I put it underneath the visual_guidelines.scss for some reason. Paul tried to help me but we both were stumped. ... maybe someone else can help.